requirements/uninstall-requirements.txt

```diff
@@ -1,2 +1,3 @@
+guppy
```

Comment:
This is probably unnecessary, maybe even undesirable for devs who have it installed and want to use it?